src/gsi/obsmod.F90

```diff
@@ -788,6 +788,14 @@ subroutine init_obsmod_dflts
     static_gsi_nopcp_dbz=0.0_r_kind
     rmesh_dbz=2
     rmesh_vr=2
+!  pmot_dbz values of 0,1,2,3 will save different sets of obs output
```

Comment:
@jderber-NOAA  Thanks again for so many new functions/optimization in this PR.  However, a concern I have is that we lack regression tests for so many changes, Though, here,  pmot_dbz is hardwired to 0 , other options 1-3 (and, hence the changes for them) will not be used.  If allowed, I would suggest to put those changes of  options of 1,2,3 in read_dbz_nc  into another future PR while we could have some verification runs for them.  We may discuss more on this off here.

Reply:
These changes actually remove some bugs in the original code (e.g., in read_radar, etc. ) where the observations were not returned to the order that they were read in.  I have tested some of the options (I agree to test all the combination of options is difficult).  I don't see where the code could use a different set of observations and thus would not produce a different analysis.  What is more difficult to test is to whether or not the diagnostic files are always correct.  I think they are, but cannot be absolutely sure without running a lot of tests.

I think it is better to get these changes in and if there are problems with the diagnostic files, they can be fixed easily.

Reply:
Thanks for the clarification.  I agree.